docs/syntax/file_inclusion.md

```diff
@@ -6,6 +6,10 @@ File inclusion is useful for
 
 If there are portions of content that are relevant to multiple pages, you can inject content from another file into any given MD file using the `include` directive.
 
+:::{note}
+Files to be included must live in a `_snippets` folder otherwise the docs builder will get mad. This folder can live anywhere. 
```

Comment:
```suggestion
Files to be included must live in a `_snippets` to be considered a snippet file. This folder can live anywhere. 
```

I love the idea of the docs-builder getting mad, but these are just the rules it has laid out for y'all and it obeys them without emotion 😸

Reply:
🤣